fido/conf/versions.xml

```diff
@@ -2,7 +2,7 @@
 <versions>
 	<pronomVersion>94</pronomVersion>
 	<pronomSignature>formats-v94.xml</pronomSignature>
-	<pronomContainerSignature>container-signature-20180917.xml</pronomContainerSignature>
+	<pronomContainerSignature>container-signature-20160121.xml</pronomContainerSignature>
```

Comment:
Is this the right version? This looks like a downgrade.

Reply:
Whoops -- the comment in updatesignatures.py#37 let me know this is hardcoded and has to be manually changed, and I missed it! Thank you!